silta-node/18-alpine/entrypoint.sh

```diff
@@ -41,14 +44,16 @@ EOF
     env > /etc/environment
     addgroup www-admin
```

Comment:
This `addgroup` line was removed from all other versions, should be also removed from this one, right?

Reply:
absolutely, thanks for spotting that! This one is fixed now!